### DISCUSSION TITLE
Microsoft and GitHub

### DISCUSSION THREAD

**Discussion by grokk** [larrykollar/tines#21] [General]
The time has come to speak of many things... specifically here: what a project like tines should be doing about the acquisition of (corporate) GitHub -- the de facto center of the FOSS/Free/Libre software repository Universe -- by (Embrace. Extend. Extinguish/Exterminate) Microsoft.

And for the record: I (for one) will object to anyone characterizing Microsoft and its plans for 'Open Source' as anything other than duplicitous -- however sweetly the siren song 'charm offensive' is, for the foreseeable future...

**Comment by larrykollar** [comment]
For now, what I'm doing is keeping an eye on things. While I myself am skeptical, people in the FOSS community with far higher standings than mine have said this isn't Ballmer's MS (see LWN.net for quotes)… of course, it doesn't take much to have a higher standing than mine in the FOSS community. :)

Besides, even if MS was inclined to throw away enough money to buy housing for >7,000 families (using figures local to my exurban/rural area), and shut down Github.com tomorrow, something new would spring up to take its place over the weekend. IIRC, the GitHub software is Free, right? If not, I could always give a resigned sigh and move to SourceForge (which was also problematic under its previous leadership).

**Comment by grokk** [comment]
'people in the FOSS community with far higher standings than mine have said this isn't Ballmer's MS'

That's _exactly_ the sort of FUD I was referring to re the 'charm offensive'. And for the record: I've been involved in hardcore politix for a long, long time -- AFAIC longer, and more deeply than most anyone you refer to -- and I _KNOW_ M$ is up to NO good. They have a Plan. How could they not. They ARE playing a 'long game' with FOSS.

But indeed -- there's no hurry here, as you said. Just that the subject must be broached, now.
